docs/source/ledger/ledger.md

```diff
@@ -375,7 +375,7 @@ the same chaincode can access a given namespace.
 
 A blockchain is not namespaced. It contains transactions from many different
 smart contract namespaces. You can read more about chaincode namespaces in this
-[topic](./developapps/chaincodenamespace.html).
+[topic](../developapps/chaincodenamespace.html).
```

Comment:
Can you fix this sentence while you are in here -> `A blockchain is not namespaced.`

to
A blockchain is not a namespace.

Reply:
I don't love the original text "A blockchain is not namespaced". But I don't think the suggestion "A blockchain is not a namespace" improves upon it.
So for now I suggest we merge the link fixes since that was the intent of this PR, and leave the other sentence as it was.